ProcessLib/LIE/SmallDeformation/LocalAssembler/SmallDeformationLocalAssemblerInterface.h

```diff
@@ -90,6 +90,21 @@ class SmallDeformationLocalAssemblerInterface
             "is not implemented");
     }
 
+    void computeSecondaryVariableConcrete(
+        const double t, std::vector<double> const& local_x_) override
+    {
+        if (!_dofIndex_to_localIndex.empty())
+        {
+            auto const local_dof_size = local_x_.size();
```

Comment:
You can directly use local_x_.size() in the following for loop.

Reply:
Replaced accordingly.